internal/config/toml.go

```diff
@@ -170,11 +170,26 @@ func (t *Toml) Get(key string) interface{} {
 	return (*toml.Tree)(t).Get(key)
 }
 
+// GetDefault returns the value for the specified key and falls back to the default value if the Get call fails
+func (t *Toml) GetDefault(key string, def interface{}) interface{} {
+	val := t.Get(key)
+	if val == nil {
+		return def
+	}
+	return val
```

Comment:
Is there a reason that the following doesn't work:
```suggestion
	return (*toml.Tree)(t).GetDefault(key, def)
```